src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -544,6 +548,32 @@ void testPartialUpdateValidationNoUpdate() {
         assertFalse(errors.hasErrors());
     }
 
+    @Test
+    void testPartialRemoveValidationNoRemoveStatement() {
+        setIsRoeUpdateEnabledFeatureFlag(true);
+        buildPartialOverseasEntityRemoveSubmissionDto();
+
+        Errors errors = overseasEntitySubmissionDtoValidator.validatePartial(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+
+        assertFalse(errors.hasErrors());
+        verify(removeValidator, never()).validate(any(), any(), any());
+    }
+
+    @Test
+    void testPartialRemoveValidationRemoveStatementPresent() {
+        setIsRoeUpdateEnabledFeatureFlag(true);
+        buildPartialOverseasEntityRemoveSubmissionDto();
+
+        RemoveDto removeDto = new RemoveDto();
+        removeDto.setIsNotProprietorOfLand(true);
```

Comment:
Could have a constructor that takes `isNotProprietorOfLand` as an argument to make this more concise. Not a deal breaker.

Reply:
Actually, this is just used in a test class so ignore.

Reply:
ignored